prime-router/docs/universal-pipeline/route.md

```diff
@@ -1,17 +1,462 @@
+# Context
 
-Note: This step occurs after the Convert step and before the Translate step.
+The Route function’s purpose is to match FHIR bundles with receivers. Each receiver connected with ReportStream has unique interests in the data that flows through the pipeline. Routing is designed to find the data that meet those interests.
 
-## Filters
+The Route function follows the convert function. At this point all data will be in FHIR format. These messages are passed to the FHIR Router which first decodes creating a FHIR Bundle. **_FHIRRouter.applyFilters_** does the work to find receivers that accept the bundle. With the list of acceptable receivers, FHIR Endpoints are added to the Provenance resource identifying those receivers.  An endpoint describes the details of a receiver including which test results to include. With that information, the message is passed to the Translate function where receiver specific work is done.
 
-Make sure to explain the default for each filter - explain that routing in RS depends on filters. If a receiver wants a message to go to them, it has to pass their filters. Additionally, the default filter also depends on the associated topic. (Already some data on this, needs updating)
-[Reference](https://docs.google.com/spreadsheets/d/1xAfkzPqs4yA3tAdXW4JShtENTZW_kSW6rlH_Hjx-glI/edit#gid=0) for user settings requested upon onboarding
 
-- Quality Filter
-- RoutingFilter
-- JurisdictionalFilter
-- Processing Mode Filter
-- Condition Filter
-- Developer section on how they work with examples. Also explain in this section how they can now be put on separate lines. 
+## Topic
+
+A Topic must be set for all senders and receivers. The choice of topic determines which pipeline is used (Universal or Covid) and will affect how routing takes place. The routing step will start by limiting available receivers to only those with a topic matching the sender topic. Topics include:
+
+
+<table>
+  <tr>
+   <td><strong>Topic Name</strong>
+   </td>
+   <td><strong>Pipeline</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>full-elr
+   </td>
+   <td>Universal
+   </td>
+   <td>General use for test result reporting
+   </td>
+  </tr>
+  <tr>
+   <td>etor-ti
+   </td>
+   <td>Universal
+   </td>
+   <td>ETOR project with Flexion’s Trusted Intermediary
+   </td>
+  </tr>
+  <tr>
+   <td>elr-elims
+   </td>
+   <td>Universal
+   </td>
+   <td>ELR project with ELIMS
+   </td>
+  </tr>
+</table>
+
+# FHIRPath
+
+Although existing in the COVID Pipeline, filter functions do not exist in the Universal Pipeline. The functionality is covered by [FHIRPath](https://build.fhir.org/fhirpath.html). Paths are defined using [FHIRPath](http://hl7.org/fhirpath/), which is an expression language defined by FHIR. In its simplest form, this can take the form of a single dotted path: `Bundle.entry.resource.ofType(Patient).name.family`.` `
```

Comment:
I think we should leave out the piece about filter functions if they do not exist in UP.